packages/driver/test/cypress/integration/commands/assertions_spec.coffee

```diff
@@ -626,6 +628,243 @@ describe "src/cy/commands/assertions", ->
+      obj1 = Array(500).fill().map((x, i) ->
+        if i % 100 is 0
+          return 'good'
+        return i
+      )
+      obj2 = Array(500).fill().map((x, i) ->
+        if i % 100 is 0
+          return 'bad'
+        return i
+      )
+      cy.on "log:added", (attrs, log) =>
+        if attrs.name is "assert"
+          { consoleProps } = attrs
+          
+          cy.removeAllListeners("log:added")
+
+          expect(consoleProps.Diff.split('\n')).to.have.length.lt(60)
+          done()
+      try
+        expect(obj2).to.eq(obj1)
+      catch err
+
+    it "consoleProps error diffs are logged properly", (done) ->
+      obj1 = 'good'
+      obj2 = 'bad'
+      cy.once "log:added", (attrs, log) =>
+        if attrs.name is "assert"
+          { consoleProps } = attrs
+
+          cy.removeAllListeners("log:added")
+
+          expect(logger._formatConsoleDiff(consoleProps.Diff)).to.deep.eq([
+            " %c\n   -bad %c\n   +good %c\n   ",
+            "color:red",
+            "color:green",
+            ""
+          ])
+          done()
+      expect(obj2).to.eq(obj1)
+
+    ## TODO: use snapshot here
+    it "consoleProps errors with diffs are logged properly and escaped", (done) ->
+      stub = cy.spy().as('console')
+      cy.stub(console, 'log').callsFake(stub)
+      cy.stub(console, 'groupCollapsed').callsFake(stub)
+      obj1 = ['%cfo%co%c', '%%cf%coo%%%c']
+      obj2 = ['%cfo%co%c', '%%cf%coo%%c']
+
+      cy.once "log:added", (attrs, log) =>
+        { consoleProps } = attrs
+        logger._logValues(consoleProps)
+        expect(stub.args).to.deep.eq([
+          [
+            "%cCommand:  ",
+            "font-weight: bold",
+            "assert"
+          ],
+          [
+            "%cActual:   ",
+            "font-weight: bold",
+            [
+              "%cfo%co%c",
+              "%%cf%coo%%c"
+            ]
+          ],
+          [
+            "%cExpected: ",
+            "font-weight: bold",
+            [
+              "%cfo%co%c",
+              "%%cf%coo%%%c"
+            ]
+          ],
+          [
+            "%cMessage:  ",
+            "font-weight: bold",
+            "expected [ %cfo%co%c, %%cf%coo%%c ] to equal [ %cfo%co%c, %%cf%coo%%%c ]"
+          ],
+          [
+            "%cError:    ",
+            "font-weight: bold",
+            "AssertionError: expected [ '%cfo%co%c', '%%cf%coo%%c' ] to equal [ '%cfo%co%c', '%%cf%coo%%%c' ]"
+          ],
+          [
+            "%cDiff:  %c+ expected %c- actual",
+            "",
+            "color:green",
+            "color:red"
+          ],
+          [
+            "  %c\n    [ %c\n      \"%%cfo%%co%%c\" %c\n   -  \"%%%%cf%%coo%%%%c\" %c\n   +  \"%%%%cf%%coo%%%%%%c\" %c\n    ] %c\n   ",
+            "",
+            "",
+            "color:red",
+            "color:green",
+            "",
+            ""
+          ]
+        ])
+        done()
+
+      expect(obj2).to.eq(obj1)
+    
+    ## TODO: use a snapshot here
```

Comment:
do you want to still use a snapshot here?

Reply:
dont think it's worth the hassle right now since we don't have any other snapshot tests in the driver